plugin/trino-base-jdbc/src/test/java/io/trino/plugin/jdbc/TestingH2JdbcClient.java

```diff
@@ -104,7 +104,7 @@ public TestingH2JdbcClient(BaseJdbcConfig config, ConnectionFactory connectionFa
     }
 
     @Override
-    public Collection<String> listSchemas(Connection connection)
+    public Set<String> listSchemas(Connection connection)
```

Comment:
Can we avoid changing this method signature or do it in separate preparatory commit?

Reply:
refactored